python/README.md

```diff
@@ -1,13 +0,0 @@
-# notcurses
```

Comment:
This doesn't seem to have been replaced by .readthedocs.yml or anything else. Updating the author is certainly called for, but ought this go away entirely? Isn't it used by pypi?